front/app/components/SignUpIn/SignUp/Consent.tsx

```diff
@@ -16,13 +17,12 @@ import { fontSizes } from 'utils/styleUtils';
 
 const Container = styled.div`
   display: flex;
+  gap: 15px;
   flex-direction: column;
   align-items: stretch;
 `;
 
-const CheckboxWrapper = styled.div`
-  margin-bottom: 15px;
-`;
+const CheckboxWrapper = styled.div``;
```

Comment:
Not sure if this is needed at all. First time dipping into styled components.

Reply:
You can delete it both here and in the component, or otherwise use a `Box`

Reply:
Done that, thanks Luuc! :)